### PR TITLE
python3Packages.graphqlclient: init at 0.2.4

### PR DESCRIPTION
**Pull request by lde**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [x] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 130261` at 087972fe run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.graphqlclient</li>
    <li>python39Packages.graphqlclient</li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md">maintainers-missing</a>

Package does not have a maintainer. Consider adding yourself?

Near pkgs/development/python-modules/graphqlclient/default.nix:19:3:
```
   |
19 |   meta = with lib; {
   |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-python-tests.md">no-python-tests</a>

Test runner could not discover any test cases: ‘Ran 0 tests in 0.000s’
Near pkgs/development/python-modules/graphqlclient/default.nix:20:0:
```
   |
20 |     description = "Simple GraphQL client for Python 2.7+";
   | ^
```
</li>
  </ul>
</details>

File: pkgs/development/python-modules/graphqlclient/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    sha256 = "0b6r3ng78qsn7c9zksx4rgdkmp5296d40kbmjn8q614cz0ymyc5k";
+  };
+  checkInputs = [
+      six
+    ];
+
+  meta = with lib; {
+    description = "Simple GraphQL client for Python 2.7+";
+    homepage = "https://github.com/prisma-labs/python-graphql-client";
+    license = lib.licenses.mit;
```
Comment (fabaff) [inline pkgs/development/python-modules/graphqlclient/default.nix:22]:
```suggestion
    license = licenses.mit;
```

File: pkgs/development/python-modules/graphqlclient/default.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0b6r3ng78qsn7c9zksx4rgdkmp5296d40kbmjn8q614cz0ymyc5k";
+  };
+  checkInputs = [
+      six
+    ];
+
+  meta = with lib; {
+    description = "Simple GraphQL client for Python 2.7+";
```
Comment (fabaff) [inline pkgs/development/python-modules/graphqlclient/default.nix:20]: I don't think that version details add value in the description.

```suggestion
    description = "Simple GraphQL client for Python";
```

File: pkgs/development/python-modules/graphqlclient/default.nix [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+buildPythonPackage rec {
+  pname = "graphqlclient";
+  version = "0.2.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0b6r3ng78qsn7c9zksx4rgdkmp5296d40kbmjn8q614cz0ymyc5k";
+  };
+  checkInputs = [
```
Comment (fabaff) [inline pkgs/development/python-modules/graphqlclient/default.nix:15]: `six` is a run-time [requirement](https://github.com/prisma-labs/python-graphql-client/blob/master/setup.py#L16-L17).

```suggestion

  propagatedBuildInputs = [
```

File: pkgs/development/python-modules/graphqlclient/default.nix [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+      six
+    ];
+
```
Comment (fabaff) [inline pkgs/development/python-modules/graphqlclient/default.nix:18]: Don't run tests but add `pythonImportsCheck`

```suggestion
  # Project has no tests
  doCheck = false;

  pythonImportsCheck = [ "graphqlclient" ];
```

**Comment by fabaff** [comment]
> Package does not have a maintainer. Consider adding yourself?

New packages need to have a maintainer.

File: pkgs/development/python-modules/graphqlclient/default.nix [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  pname = "graphqlclient";
+  version = "0.2.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0b6r3ng78qsn7c9zksx4rgdkmp5296d40kbmjn8q614cz0ymyc5k";
+  };
+  propagatedBuildInputs = [
+      six
+    ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/graphqlclient/default.nix:17]:
```suggestion
  };

  propagatedBuildInputs = [
    six
  ];
```

File: pkgs/development/python-modules/graphqlclient/default.nix [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+  # Project has no tests
+  doCheck = false;
+
+  pythonImportsCheck = [ "graphqlclient" ];
+
+  meta = with lib; {
+    description = "Simple GraphQL client for Python";
+    homepage = "https://github.com/prisma-labs/python-graphql-client";
+    license = licenses.mit;
+    maintainers = with maintainers; [ lde ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/graphqlclient/default.nix:29]: Please add yourself as a maintainer by adding an alphabetically sorted entry to `maintainers/maintainer-list.nix` and commit this change into a separate commit with the message ``maintainers: add NAME`` into this PR.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Also please squash the package related commits together.

File: maintainers/maintainer-list.nix [hunk 7]
```diff
@@ -5876,6 +5876,12 @@
     githubId = 1104419;
     name = "Lucas Hoffmann";
   };
+  lde = {
+    email = "lilian.deloche@puck.fr";
+    github = "lde"
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:5881]:
```suggestion
    github = "lde";
```

Comment (lde) [reply maintainers/maintainer-list.nix:5881]: sorry typo

**Pull request merged**